msg/json/en.json

```diff
@@ -1,7 +1,7 @@
 {
 	"@metadata": {
 		"author": "Ellen Spertus <ellen.spertus@gmail.com>",
-		"lastupdated": "2015-10-29 13:26:30.538106",
+		"lastupdated": "2016-09-26 18:18:05.388794",
```

Comment:
I don't know what this is. @ruwang is this normal? Maybe because I complied the library?